README.md

```diff
@@ -104,7 +104,7 @@ The following constants are reported as the only parameter to the
 
 ### Methods
 
-- `media.getCurrentAmplitude`: Returns the current position within an audio file.
+- `media.getCurrentAmplitude`: Returns the current amplitude of the current recording.
```

Comment:
I saw the issue when you created it, but I don't think the new definition is right.

I think it should be "Returns the current amplitude within an audio file." as this is not a recording, you should also change the other occurrence of "Returns the current amplitude of the current recording"